src/Microsoft.Unity.Analyzers.Tests/ContextMenuSuppressorTests.cs

```diff
@@ -137,4 +137,4 @@ static void DoMenuItem()
 		}
 
 	}
-}
+}
```

Comment:
Unneeded change. Our editorconfig is enforcing a new line at the end of all text files if I'm not wrong